source/guides/clear/swupd.rst

```diff
@@ -344,7 +344,7 @@ Related topics
 
 .. _swupd source documentation: https://github.com/clearlinux/swupd-client/blob/master/docs/swupd.1.rst
 
-.. _Kata Containers: https://clearlinux.org/downloads/containers
+.. _Kata Containers: https://clearlinux.github.io/downloads/containers
```

Comment:
I don't think this is supposed to be changed since it is going to the content part of the site. Anything under downloads probably needs to not change.